### ISSUE TITLE
[Feature suggestion] Infer parameters from control glyph

### ISSUE THREAD

**Issue by impallari** [thomgb/glyphmorfExtension#2]
Awesome project!
Here is a little feature suggestion:
Lets say I have a master of a regular font, and another file where I have draw only a single glyph /n but in condensed form (the control glyph).
It will be cool if the tool could load the "control glyph" and somehow infer the parameter needed to achieve a similar result, so one could apply those parameters to the regular master and generate an entire condensed font.

Basically: Instead of manually selecting the parameters, we could manually draw what we need to achieve in just one glyph and translate that to a newly generated font.
